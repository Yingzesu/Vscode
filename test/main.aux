\relax 
\providecommand{\transparent@use}[1]{}
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{nav}{\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {1}{1}}}
\@writefile{nav}{\headcommand {\slideentry {0}{0}{2}{2/2}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {2}{2}}}
\@writefile{toc}{\beamer@sectionintoc {1}{Why the S-matrix is so important?}{3}{0}{1}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {1}{2}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {1}{2}}}
\@writefile{nav}{\headcommand {\sectionentry {1}{Why the S-matrix is so important?}{3}{Why the S-matrix is so important?}{0}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{1}{3/3}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {3}{3}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{2}{4/7}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {4}{7}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{3}{8/8}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {8}{8}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{4}{9/9}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {9}{9}}}
\@writefile{toc}{\beamer@sectionintoc {2}{CHY Formula}{10}{0}{2}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {3}{9}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {3}{9}}}
\@writefile{nav}{\headcommand {\sectionentry {2}{CHY Formula}{10}{CHY Formula}{0}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{1}{10/10}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {10}{10}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{2}{11/11}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {11}{11}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{3}{12/12}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {12}{12}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{4}{13/13}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {13}{13}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{5}{14/15}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {14}{15}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{6}{16/17}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {16}{17}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{7}{18/18}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {18}{18}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{8}{19/20}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {19}{20}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{9}{21/21}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {21}{21}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{10}{22/22}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {22}{22}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{11}{23/24}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {23}{24}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{12}{25/25}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {25}{25}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{13}{26/27}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {26}{27}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{14}{28/28}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {28}{28}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{15}{29/29}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {29}{29}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{16}{30/30}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {30}{30}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{17}{31/31}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {31}{31}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{18}{32/32}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {32}{32}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{19}{33/33}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {33}{33}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{20}{34/34}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {34}{34}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{21}{35/35}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {35}{35}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{22}{36/36}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {36}{36}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{23}{37/38}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {37}{38}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{24}{39/39}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {39}{39}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{25}{40/40}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {40}{40}}}
\@writefile{toc}{\beamer@sectionintoc {3}{Summary and Outlook}{41}{0}{3}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {10}{40}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {10}{40}}}
\@writefile{nav}{\headcommand {\sectionentry {3}{Summary and Outlook}{41}{Summary and Outlook}{0}}}
\@writefile{nav}{\headcommand {\slideentry {3}{0}{1}{41/41}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {41}{41}}}
\@writefile{nav}{\headcommand {\slideentry {3}{0}{2}{42/42}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {42}{42}}}
\@writefile{nav}{\headcommand {\slideentry {3}{0}{3}{43/43}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {43}{43}}}
\@writefile{nav}{\headcommand {\slideentry {3}{0}{4}{44/44}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {44}{44}}}
\@writefile{toc}{\beamer@sectionintoc {4}{Appendix}{45}{0}{4}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {41}{44}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {41}{44}}}
\@writefile{nav}{\headcommand {\sectionentry {4}{Appendix}{45}{Appendix}{0}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{1}{45/45}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {45}{45}}}
\global\@namedef{mainendframenumber}{36}
\@writefile{nav}{\headcommand {\gdef \insertmainframenumber {36}}}
\@writefile{nav}{\headcommand {\partentry {\translate  {Appendix}}{1}}}
\@writefile{nav}{\headcommand {\beamer@partpages {1}{45}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {45}{45}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {45}{45}}}
\@writefile{nav}{\headcommand {\beamer@appendixpages {46}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{2}{46/46}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {46}{46}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{3}{47/47}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {47}{47}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{4}{48/48}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {48}{48}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{5}{49/49}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {49}{49}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{6}{50/50}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {50}{50}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{7}{51/51}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {51}{51}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{8}{52/52}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {52}{52}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{9}{53/53}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {53}{53}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{10}{54/54}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {54}{54}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{11}{55/55}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {55}{55}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{12}{56/56}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {56}{56}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{13}{57/57}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {57}{57}}}
\@writefile{snm}{\beamer@slide {jacobbi}{58}}
\newlabel{jacobbi}{{13}{58}{\appendixname }{Doc-Start}{}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{14}{58/58}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {58}{58}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{15}{59/59}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {59}{59}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{16}{60/60}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {60}{60}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{17}{61/61}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {61}{61}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{18}{62/62}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {62}{62}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{19}{63/63}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {63}{63}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{20}{64/64}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {64}{64}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{21}{65/65}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {65}{65}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{22}{66/66}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {66}{66}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{23}{67/67}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {67}{67}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{24}{68/68}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {68}{68}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{25}{69/69}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {69}{69}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{26}{70/70}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {70}{70}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{27}{71/71}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {71}{71}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{28}{72/72}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {72}{72}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{29}{73/73}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {73}{73}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{30}{74/74}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {74}{74}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{31}{75/76}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {75}{76}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{32}{77/77}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {77}{77}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{33}{78/78}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {78}{78}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{34}{79/79}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {79}{79}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{35}{80/80}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {80}{80}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{36}{81/81}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {81}{81}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{37}{82/82}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {82}{82}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{38}{83/83}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {83}{83}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{39}{84/84}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {84}{84}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{40}{85/85}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {85}{85}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{41}{86/86}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {86}{86}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{42}{87/88}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {87}{88}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{43}{89/89}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {89}{89}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{44}{90/90}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {90}{90}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{45}{91/91}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {91}{91}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{46}{92/92}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {92}{92}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{47}{93/93}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {93}{93}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{48}{94/94}{}{1}}}
\@writefile{nav}{\headcommand {\beamer@framepages {94}{94}}}
\@writefile{nav}{\headcommand {\beamer@partpages {46}{94}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {46}{94}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {46}{94}}}
\@writefile{nav}{\headcommand {\beamer@documentpages {94}}}
\@writefile{nav}{\headcommand {\gdef \inserttotalframenumber {47}}}
\global\@namedef{appendixtotalframenumber}{47}
\global\@namedef{inserttotalframenumber}{36}
\@writefile{nav}{\headcommand {\gdef \inserttotalframenumber {36}}}
\@writefile{nav}{\headcommand {\gdef \appendixtotalframenumber {47}}}
\gdef\svg@ink@ver@settings{{\m@ne }{inkscape}{\m@ne }}
\gdef \@abspage@last{94}
