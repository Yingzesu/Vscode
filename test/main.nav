\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}
\headcommand {\beamer@framepages {1}{1}}
\headcommand {\slideentry {0}{0}{2}{2/2}{}{0}}
\headcommand {\beamer@framepages {2}{2}}
\headcommand {\beamer@sectionpages {1}{2}}
\headcommand {\beamer@subsectionpages {1}{2}}
\headcommand {\sectionentry {1}{Why the S-matrix is so important?}{3}{Why the S-matrix is so important?}{0}}
\headcommand {\slideentry {1}{0}{1}{3/3}{}{0}}
\headcommand {\beamer@framepages {3}{3}}
\headcommand {\slideentry {1}{0}{2}{4/7}{}{0}}
\headcommand {\beamer@framepages {4}{7}}
\headcommand {\slideentry {1}{0}{3}{8/8}{}{0}}
\headcommand {\beamer@framepages {8}{8}}
\headcommand {\slideentry {1}{0}{4}{9/9}{}{0}}
\headcommand {\beamer@framepages {9}{9}}
\headcommand {\beamer@sectionpages {3}{9}}
\headcommand {\beamer@subsectionpages {3}{9}}
\headcommand {\sectionentry {2}{CHY Formula}{10}{CHY Formula}{0}}
\headcommand {\slideentry {2}{0}{1}{10/10}{}{0}}
\headcommand {\beamer@framepages {10}{10}}
\headcommand {\slideentry {2}{0}{2}{11/11}{}{0}}
\headcommand {\beamer@framepages {11}{11}}
\headcommand {\slideentry {2}{0}{3}{12/12}{}{0}}
\headcommand {\beamer@framepages {12}{12}}
\headcommand {\slideentry {2}{0}{4}{13/13}{}{0}}
\headcommand {\beamer@framepages {13}{13}}
\headcommand {\slideentry {2}{0}{5}{14/15}{}{0}}
\headcommand {\beamer@framepages {14}{15}}
\headcommand {\slideentry {2}{0}{6}{16/17}{}{0}}
\headcommand {\beamer@framepages {16}{17}}
\headcommand {\slideentry {2}{0}{7}{18/18}{}{0}}
\headcommand {\beamer@framepages {18}{18}}
\headcommand {\slideentry {2}{0}{8}{19/20}{}{0}}
\headcommand {\beamer@framepages {19}{20}}
\headcommand {\slideentry {2}{0}{9}{21/21}{}{0}}
\headcommand {\beamer@framepages {21}{21}}
\headcommand {\slideentry {2}{0}{10}{22/22}{}{0}}
\headcommand {\beamer@framepages {22}{22}}
\headcommand {\slideentry {2}{0}{11}{23/24}{}{0}}
\headcommand {\beamer@framepages {23}{24}}
\headcommand {\slideentry {2}{0}{12}{25/25}{}{0}}
\headcommand {\beamer@framepages {25}{25}}
\headcommand {\slideentry {2}{0}{13}{26/27}{}{0}}
\headcommand {\beamer@framepages {26}{27}}
\headcommand {\slideentry {2}{0}{14}{28/28}{}{0}}
\headcommand {\beamer@framepages {28}{28}}
\headcommand {\slideentry {2}{0}{15}{29/29}{}{0}}
\headcommand {\beamer@framepages {29}{29}}
\headcommand {\slideentry {2}{0}{16}{30/30}{}{0}}
\headcommand {\beamer@framepages {30}{30}}
\headcommand {\slideentry {2}{0}{17}{31/31}{}{0}}
\headcommand {\beamer@framepages {31}{31}}
\headcommand {\slideentry {2}{0}{18}{32/32}{}{0}}
\headcommand {\beamer@framepages {32}{32}}
\headcommand {\slideentry {2}{0}{19}{33/33}{}{0}}
\headcommand {\beamer@framepages {33}{33}}
\headcommand {\slideentry {2}{0}{20}{34/34}{}{0}}
\headcommand {\beamer@framepages {34}{34}}
\headcommand {\slideentry {2}{0}{21}{35/35}{}{0}}
\headcommand {\beamer@framepages {35}{35}}
\headcommand {\slideentry {2}{0}{22}{36/36}{}{0}}
\headcommand {\beamer@framepages {36}{36}}
\headcommand {\slideentry {2}{0}{23}{37/38}{}{0}}
\headcommand {\beamer@framepages {37}{38}}
\headcommand {\slideentry {2}{0}{24}{39/39}{}{0}}
\headcommand {\beamer@framepages {39}{39}}
\headcommand {\slideentry {2}{0}{25}{40/40}{}{0}}
\headcommand {\beamer@framepages {40}{40}}
\headcommand {\beamer@sectionpages {10}{40}}
\headcommand {\beamer@subsectionpages {10}{40}}
\headcommand {\sectionentry {3}{Summary and Outlook}{41}{Summary and Outlook}{0}}
\headcommand {\slideentry {3}{0}{1}{41/41}{}{0}}
\headcommand {\beamer@framepages {41}{41}}
\headcommand {\slideentry {3}{0}{2}{42/42}{}{0}}
\headcommand {\beamer@framepages {42}{42}}
\headcommand {\slideentry {3}{0}{3}{43/43}{}{0}}
\headcommand {\beamer@framepages {43}{43}}
\headcommand {\slideentry {3}{0}{4}{44/44}{}{0}}
\headcommand {\beamer@framepages {44}{44}}
\headcommand {\beamer@sectionpages {41}{44}}
\headcommand {\beamer@subsectionpages {41}{44}}
\headcommand {\sectionentry {4}{Appendix}{45}{Appendix}{0}}
\headcommand {\slideentry {4}{0}{1}{45/45}{}{0}}
\headcommand {\beamer@framepages {45}{45}}
\headcommand {\gdef \insertmainframenumber {36}}
\headcommand {\partentry {\translate {Appendix}}{1}}
\headcommand {\beamer@partpages {1}{45}}
\headcommand {\beamer@sectionpages {45}{45}}
\headcommand {\beamer@subsectionpages {45}{45}}
\headcommand {\beamer@appendixpages {46}}
\headcommand {\slideentry {4}{0}{2}{46/46}{}{1}}
\headcommand {\beamer@framepages {46}{46}}
\headcommand {\slideentry {4}{0}{3}{47/47}{}{1}}
\headcommand {\beamer@framepages {47}{47}}
\headcommand {\slideentry {4}{0}{4}{48/48}{}{1}}
\headcommand {\beamer@framepages {48}{48}}
\headcommand {\slideentry {4}{0}{5}{49/49}{}{1}}
\headcommand {\beamer@framepages {49}{49}}
\headcommand {\slideentry {4}{0}{6}{50/50}{}{1}}
\headcommand {\beamer@framepages {50}{50}}
\headcommand {\slideentry {4}{0}{7}{51/51}{}{1}}
\headcommand {\beamer@framepages {51}{51}}
\headcommand {\slideentry {4}{0}{8}{52/52}{}{1}}
\headcommand {\beamer@framepages {52}{52}}
\headcommand {\slideentry {4}{0}{9}{53/53}{}{1}}
\headcommand {\beamer@framepages {53}{53}}
\headcommand {\slideentry {4}{0}{10}{54/54}{}{1}}
\headcommand {\beamer@framepages {54}{54}}
\headcommand {\slideentry {4}{0}{11}{55/55}{}{1}}
\headcommand {\beamer@framepages {55}{55}}
\headcommand {\slideentry {4}{0}{12}{56/56}{}{1}}
\headcommand {\beamer@framepages {56}{56}}
\headcommand {\slideentry {4}{0}{13}{57/57}{}{1}}
\headcommand {\beamer@framepages {57}{57}}
\headcommand {\slideentry {4}{0}{14}{58/58}{}{1}}
\headcommand {\beamer@framepages {58}{58}}
\headcommand {\slideentry {4}{0}{15}{59/59}{}{1}}
\headcommand {\beamer@framepages {59}{59}}
\headcommand {\slideentry {4}{0}{16}{60/60}{}{1}}
\headcommand {\beamer@framepages {60}{60}}
\headcommand {\slideentry {4}{0}{17}{61/61}{}{1}}
\headcommand {\beamer@framepages {61}{61}}
\headcommand {\slideentry {4}{0}{18}{62/62}{}{1}}
\headcommand {\beamer@framepages {62}{62}}
\headcommand {\slideentry {4}{0}{19}{63/63}{}{1}}
\headcommand {\beamer@framepages {63}{63}}
\headcommand {\slideentry {4}{0}{20}{64/64}{}{1}}
\headcommand {\beamer@framepages {64}{64}}
\headcommand {\slideentry {4}{0}{21}{65/65}{}{1}}
\headcommand {\beamer@framepages {65}{65}}
\headcommand {\slideentry {4}{0}{22}{66/66}{}{1}}
\headcommand {\beamer@framepages {66}{66}}
\headcommand {\slideentry {4}{0}{23}{67/67}{}{1}}
\headcommand {\beamer@framepages {67}{67}}
\headcommand {\slideentry {4}{0}{24}{68/68}{}{1}}
\headcommand {\beamer@framepages {68}{68}}
\headcommand {\slideentry {4}{0}{25}{69/69}{}{1}}
\headcommand {\beamer@framepages {69}{69}}
\headcommand {\slideentry {4}{0}{26}{70/70}{}{1}}
\headcommand {\beamer@framepages {70}{70}}
\headcommand {\slideentry {4}{0}{27}{71/71}{}{1}}
\headcommand {\beamer@framepages {71}{71}}
\headcommand {\slideentry {4}{0}{28}{72/72}{}{1}}
\headcommand {\beamer@framepages {72}{72}}
\headcommand {\slideentry {4}{0}{29}{73/73}{}{1}}
\headcommand {\beamer@framepages {73}{73}}
\headcommand {\slideentry {4}{0}{30}{74/74}{}{1}}
\headcommand {\beamer@framepages {74}{74}}
\headcommand {\slideentry {4}{0}{31}{75/76}{}{1}}
\headcommand {\beamer@framepages {75}{76}}
\headcommand {\slideentry {4}{0}{32}{77/77}{}{1}}
\headcommand {\beamer@framepages {77}{77}}
\headcommand {\slideentry {4}{0}{33}{78/78}{}{1}}
\headcommand {\beamer@framepages {78}{78}}
\headcommand {\slideentry {4}{0}{34}{79/79}{}{1}}
\headcommand {\beamer@framepages {79}{79}}
\headcommand {\slideentry {4}{0}{35}{80/80}{}{1}}
\headcommand {\beamer@framepages {80}{80}}
\headcommand {\slideentry {4}{0}{36}{81/81}{}{1}}
\headcommand {\beamer@framepages {81}{81}}
\headcommand {\slideentry {4}{0}{37}{82/82}{}{1}}
\headcommand {\beamer@framepages {82}{82}}
\headcommand {\slideentry {4}{0}{38}{83/83}{}{1}}
\headcommand {\beamer@framepages {83}{83}}
\headcommand {\slideentry {4}{0}{39}{84/84}{}{1}}
\headcommand {\beamer@framepages {84}{84}}
\headcommand {\slideentry {4}{0}{40}{85/85}{}{1}}
\headcommand {\beamer@framepages {85}{85}}
\headcommand {\slideentry {4}{0}{41}{86/86}{}{1}}
\headcommand {\beamer@framepages {86}{86}}
\headcommand {\slideentry {4}{0}{42}{87/88}{}{1}}
\headcommand {\beamer@framepages {87}{88}}
\headcommand {\slideentry {4}{0}{43}{89/89}{}{1}}
\headcommand {\beamer@framepages {89}{89}}
\headcommand {\slideentry {4}{0}{44}{90/90}{}{1}}
\headcommand {\beamer@framepages {90}{90}}
\headcommand {\slideentry {4}{0}{45}{91/91}{}{1}}
\headcommand {\beamer@framepages {91}{91}}
\headcommand {\slideentry {4}{0}{46}{92/92}{}{1}}
\headcommand {\beamer@framepages {92}{92}}
\headcommand {\slideentry {4}{0}{47}{93/93}{}{1}}
\headcommand {\beamer@framepages {93}{93}}
\headcommand {\slideentry {4}{0}{48}{94/94}{}{1}}
\headcommand {\beamer@framepages {94}{94}}
\headcommand {\beamer@partpages {46}{94}}
\headcommand {\beamer@subsectionpages {46}{94}}
\headcommand {\beamer@sectionpages {46}{94}}
\headcommand {\beamer@documentpages {94}}
\headcommand {\gdef \inserttotalframenumber {47}}
\headcommand {\gdef \inserttotalframenumber {36}}
\headcommand {\gdef \appendixtotalframenumber {47}}
