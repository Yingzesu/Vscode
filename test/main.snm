\beamer@slide {jacobbi}{58}
