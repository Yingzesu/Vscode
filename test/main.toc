\beamer@sectionintoc {1}{Why the S-matrix is so important?}{3}{0}{1}
\beamer@sectionintoc {2}{CHY Formula}{10}{0}{2}
\beamer@sectionintoc {3}{Summary and Outlook}{41}{0}{3}
\beamer@sectionintoc {4}{Appendix}{45}{0}{4}
