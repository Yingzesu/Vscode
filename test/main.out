\BOOKMARK [2][]{Outline0.1}{\376\377\000W\000h\000y\000\040\000t\000h\000e\000\040\000S\000-\000m\000a\000t\000r\000i\000x\000\040\000i\000s\000\040\000s\000o\000\040\000i\000m\000p\000o\000r\000t\000a\000n\000t\000?}{}% 1
\BOOKMARK [2][]{Outline0.2}{\376\377\000C\000H\000Y\000\040\000F\000o\000r\000m\000u\000l\000a}{}% 2
\BOOKMARK [2][]{Outline0.3}{\376\377\000S\000u\000m\000m\000a\000r\000y\000\040\000a\000n\000d\000\040\000O\000u\000t\000l\000o\000o\000k}{}% 3
\BOOKMARK [2][]{Outline0.4}{\376\377\000A\000p\000p\000e\000n\000d\000i\000x}{}% 4
\BOOKMARK [1][]{Outline1}{\376\377\000A\000p\000p\000e\000n\000d\000i\000x}{}% 5
